examples/Finding Pipelines with Available Updates.ipynb

```diff
@@ -0,0 +1,790 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f3f08d9",
+   "metadata": {},
+   "source": [
+    "# Find Pipelines with Available Updates\n",
+    "\n",
+    "This script will walk you through the API calls needed to determine which pipelines, configured in a given TDP Org, have a protocol with a newver artifact version available for use.\n",
+    "\n",
+    "This script uses the following python modules\n",
+    "\n",
+    "- `requests` - for handling TDP API calls\n",
+    "- `pandas` - for data processing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "501cdf55",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "import requests\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87e98663",
+   "metadata": {},
+   "source": [
+    "### Create the authorization object\n",
+    "\n",
+    "This `auth` object is used to handle authorization for requests.  It is required for (and common to) any API usage in TDP. Therefore it is commonly recommended to save this as a `.json` file for future use."
```

Comment:
you can link to this section on doc site: https://developers.tetrascience.com/docs/set-up-your-environment-and-initialize-ts-sdk#configure-tdp-dependencies-and-authentication